packages/blade/src/utils/isReactNative/platform.all.ts

```diff
@@ -0,0 +1,32 @@
+// This `.all` file is special only for platform.ts file. Do not use anywhere else
+// This file only contains type level code and putting any runtime code will cause errors since our
+// bundler is not setup to handle this extension.
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-namespace */
+import type { Brand, IsBothFunction, NativeOrWebBrand } from './types';
+
+namespace Platform {
+  export type Name = 'web' | 'native';
+
+  // Unionify both web & native types, this will help us blade developers internally
```

Comment:
Can we be more descriptive here to explain what this does?

Reply:
Done.